data_project/README.md

```diff
@@ -28,7 +28,7 @@ no longer needed, which supports CI/CD best practices when working with Snowflak
     * what files should be included in the version creation process. In the `include_definitions` list you can:
```

Comment:
You can say that each list item is a Java Regex. Check this link for the reference: https://docs.snowflake.com/LIMITEDACCESS/data-projects/manage-data-projects#project-manifest-file